src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/OpprettBehandling/useOpprettBehandling.ts

```diff
@@ -27,6 +27,7 @@ const useOpprettBehandling = (lukkModal: () => void) => {
         behandlingstype: '',
         behandlingÅrsak: '',
     });
+    const [visÅrsakerSelect, settVisÅrsakerSelect] = useState(false);
```

Comment:
Kunne dette vært en funksjon som sjekker staten til behandlingstype?

Reply:
Ja, i stedet for å bruke state i det hele tatt?

Reply:
Jepp :)